lib/created_message_queue_handler.ts

```diff
@@ -219,6 +226,7 @@ async function createNotification(
  *          a TransientError in case of recoverable errors
  *          a PermanentError in case of unrecoverable error
  */
+// tslint:disable-next-line:cognitive-complexity
```

Comment:
🆘

Reply:
fixed. note that this come from the fact that every block ```if(...) { winston.debug }``` increases the cognitive-complexity : )